cypher-manual/clauses/match.md

```diff
@@ -404,11 +406,11 @@ Output:
 +-------+
 ```
 
-Since we don't have any node labeled Country with a property continent named *Asia*, the second `MATCH` clause returns an empty dataset and therefore the output is also an empty list. To avoid that, use `OPTIONAL MATCH` clause.
+Since the dataset doesn't contain any nodes labeled as `Country` with a property `continent` with the value `Asia`, the second `MATCH` clause returns an empty dataset and therefore the output is also an empty list. To avoid getting an empty list as an output, use `OPTIONAL MATCH` clause.
 
 ### 4.3. Using the `OPTIONAL MATCH` clause
 
-If we run the same query as before, only using the `OPTIONAL MATCH` clause: 
+To avoid getting an empty list as an output, due to any of the `MATCH` clauses returning an empty set,  use the `OPTIONAL MATCH` clause: 
```

Comment:
```suggestion
To avoid getting an empty list as an output, due to any of the `MATCH` clauses returning an empty set, use the `OPTIONAL MATCH` clause: 
```